app/controllers/page_mount_controller.rb

```diff
@@ -49,6 +50,15 @@ def domains_layout
 
   private
 
+  def store_client_type_id_in_cms_session(proxy_response)
+    # Special Hack so CC can show exclusive products for clients
+    session[:client_type_id] = JSON.parse(proxy_response.body)['client_type_id']
```

Comment:
Context: This hack is needed to keep this code working:
https://bitbucket.org/team-sealink/captain-cook-cms/src/9bf5ffc74ddb9de61b67398bd47fb5d5c360a304/vendor/extensions/captain_cook/cells/grid_cell.rb?at=master&fileviewer=file-view-default#grid_cell.rb-223